components/footer/Footer.js

```diff
@@ -97,40 +99,52 @@ const links = [
   {
     id: 1,
     title: "termsAndConditions",
-    url: "https://www.kiwi.com/en/pages/content/legal",
+    url: `https://www.kiwi.com/en/pages/content/legal${UTM_PARAMS}`,
```

Comment:
I suppose UTM_PARAMS string should not include "/?", just "param1=value1&param2=value2"